### DISCUSSION TITLE
Compressing data and storing it as .tar in a bucket

### DISCUSSION THREAD

**Discussion by rajathvan** [minio/minio#18137] [Q&A] [answered]
Hello, I need to know if its possible to copy objects from multiple buckets and create a single `.tar` file in a different bucket using APIs?

Here is the use case:
I have data in multiple buckets and would like to copy/transfer this data to a new bucket and store all of this data into a single `.tar` file. I do not want to get the data and create a tar file locally as my data might be very large and would reduce performance. I'm curious to know if there is a way to do this?

**Comment by klauspost** [comment, marked as answer]
No. You will need to do that client-side. With a properly written program you can do that fully streaming.
